src/awkward/operations/ak_concatenate.py

```diff
@@ -302,7 +308,7 @@ def action(inputs, depth, backend, **kwargs):
                 all_flatten = []
 
                 for x in nextinputs:
-                    o, f = x._offsets_and_flattened(1, 1)
+                    o, f = x._offsets_and_flattened(1, 1)  # axis=1, depth=1
```

Comment:
```suggestion
                    o, f = x._offsets_and_flattened(axis=1, depth=1)
```

Adding this comment is helpful, but calling with keyword arguments can't get out of date. This is a bit of code that was originally translated from C++, so that's why it was like this.